README.md

```diff
@@ -1,69 +1,109 @@
+
+## Joining the Community
+
+Looking to get answers to questions or engage with us in realtime? Our community is most active [on Discord](https://discord.gg/FcSNfg4). You can also ask for help on [Stack Overflow](https://stackoverflow.com/questions/tagged/fast-dna), submit requests and issues on [Github](https://github.com/Microsoft/fast-dna/issues/new/choose), or join us by contributing on [some good first issues via Github](https://github.com/Microsoft/fast-dna/labels/good%20first%20issue).
+
+We look forward to building an amazing open source community with you!
+
+## Features & benefits
+
+### Unopinionated
+
+There are a million and one great ways to build your next website or application. To support the technologies you choose, creating unopinionated code is at the center of every decision we make in FAST.
+
+This principle of being unopinionated manifests in several important ways including:
+
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components from styles and design systems to support multiple implementations without re-writing or duplicating styles. Use the design system to customize existing styled components, or build your own styles, with your design system, without having to re-build or duplicate the base components.
+* Framework agnostic tooling that lets you use our development tools with any view framework.
+* The ability to replace almost any FAST package with your package of choice. Get started with our animation package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
 
-View the [staging site](https://fast-explore-stage.azurewebsites.net/) to see unpublished changes currently staged for release in the master branch.
+### UI development and style guide tools
 
-## Documentation
+When developing components and views, excellent development tooling can make all the difference. FAST offers development and style guide tools that work with FAST components, components from other frameworks, or your components.
 
-Visit our [developer documentation](https://www.fast.design/docs/en/contributing/install) to get started.
+Try out component properties with an auto-generated property UI, get a live preview of the code based on any property’s configuration, preview localization (RTL/LTR) and themes, and preview component compositing with a transparency grid.
 
-## Troubleshooting
+Also, we built FAST development tools from re-usable packages, so if you have special needs, you can build your tools from the same shared libraries.
 
-Experiencing problems? Check our infrastructure [health status](https://www.fast.design/docs/en/contributing/status).
+### Bring your design system
 
-## Contact
```

Comment:
I would like to bring back this section.  In a marketing context, people are much less likely to read paragraphs than scan lists.  I think bullets for contacting us is important.